content/02.author-sections.md

```diff
@@ -790,6 +799,3 @@ text = parsed["content"]
 
 
 <!-- Zemin Zhang -->
-
-
-
```

Comment:
Best to not delete these lines. I should probably add something here so this does not happen.

Reply:
Fixed this in https://github.com/dhimmel/psb-manuscript/pull/3/commits/d3895a77cc155934fd34cd1086f0114557e5f900